src/backend/joanie/settings.py

```diff
@@ -548,10 +574,20 @@ class Test(Base):
     PASSWORD_HASHERS = [
         "django.contrib.auth.hashers.MD5PasswordHasher",
     ]
+    USE_SWAGGER = True
+    SWAGGER_SETTINGS = {
+        "SECURITY_DEFINITIONS": {
+            "Bearer": {
+                "type": "apiKey",
+                "name": "Authorization",
+                "in": "header",
+            }
+        }
+    }
```

Comment:
This setting seems not used anymore.
So, there is no way anymore to test api from OpenAPI UI as an authenticated user.

<img width="1515" alt="image" src="https://github.com/openfun/joanie/assets/9265241/8c2e002f-252d-4658-8975-d4c6d1904dc9">

Reply:
I removed the unused setting, but I don't get the authentication point.
Is it something we can fix later ?